server.js

```diff
@@ -0,0 +1,155 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const mongoose = require('mongoose');
+
+const User = require('./model');
+const BlogPost = require('./model');
+
+const STATUS_USER_ERROR = 422;
+const STATUS_SERVER_ERROR = 500;
+const server = express();
+
+server.use(bodyParser.json());
+
+// GET /users
+server.get('/users', (req, res) => {
+  User.find({}, (err, users) => {
+    if (err) {
+      res.status(STATUS_SERVER_ERROR);
+      res.json(err);
+    } else {
+      res.json(users);
+    }
+  });
+});
+
+// GET /users/:id
+server.get('/users/:id', (req, res) => {
+  const { id } = req.params;
+  User.findById(id, (err, user) => {
+    if (err) {
+      res.status(STATUS_SERVER_ERROR);
+      res.json(err);
+    } else if (user === null) {
+      res.json({ error: 'bear not found' });
+    } else {
+      res.json(user);
+    }
+  });
+});
+
+// POST /users
+server.post('/users', (req, res) => {
+  const { username, password } = req.body;
+  if (!username || !password) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: 'Must provide username and password' });
+    return;
+  }
+  const user = new User({ username, password });
+  user.save((err) => {
+    if (err) {
+      res.status(STATUS_SERVER_ERROR);
+      res.json(err);
+    } else {
+      res.json(user);
+    }
+  });
+});
+
+// DELETE /USERS/:id
+server.delete('/users/:id', (req, res) => {
+  const { id } = req.params;
+  if (!id) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: 'Must provide an id' });
+    return;
+  }
+  User.remove({ _id: id }, (err, removedUser) => {
```

Comment:
Because you aren't using `removedUser` anywhere in the callback function you don't need to specify a parameter for it